index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700;800&family=Poppins:ital,wght@0,600;1,600&display=swap"
+      rel="stylesheet"
+    />
+    <title>Document</title>
+  </head>
+  <body>
+    <header id="main-header">
+      <h3 id="logo">Welcome</h3>
+      <nav>
+        <button id="menu" aria-label="menu" type="button">
+          <div class="strip"></div>
+          <div class="strip"></div>
+          <div class="strip"></div>
+        </button>
+      </nav>
+    </header>
+    <section id="main-section">
+      <header>
+        <h1>
+          Hey there. I’m<br />
+          Mahdi<br />
+          <span class="green">I’m a software developer</span>
+        </h1>
+      </header>
+      <p>
+        I can help you build a product ,<br />
+        feature or website Look through<br />
+        some of my work and experience!<br />
+        If you like what you see and have<br />
+        a project you need coded,<br />
+        don’t hestiate to contact me.
+      </p>
```

Comment:
- [ ] Good job with your design. But I think it would be better you update the `text color` you used in order your application design to stick as much as possible to the one fro Figma template.
![Screenshot from 2023-02-17 04-15-45](https://user-images.githubusercontent.com/83042742/219533558-366f1cd3-1415-4949-9498-8b8e7004e9ac.png)